src/constants.ts

```diff
@@ -65,7 +65,10 @@ export interface PluginOptions {
     removeNonexistentObjects?: boolean,
     
     // Custom AWS S3 endpoint, default Amazon AWS hostname  - amazonaws.com
-    customAwsEndpointHostname?: string
+    customAwsEndpointHostname?: string,
+
+    // Override S3 website bucket hosting, for example when used with Cloudfront
```

Comment:
I think that this comment should be more descriptive, especially because we use these comments as a substitute for documentation.

Maybe something like "Disables modifications to the S3 Static Website Hosting configuration. Without S3 Static Website Hosting some features (index.html rewriting, trailing slash redirects, and serverside redirects) will not function. Not recommended, but could be useful for preventing Cloud formation Stack Drift or suppressing Terraform noise if you don't need the static website hosting functionality."